napari/_qt/qt_image_layer.py

```diff
@@ -10,7 +10,7 @@ class QtImageControls(QtLayerControls):
     def __init__(self, layer):
         super().__init__(layer)
 
-        self.layer.events.clim.connect(self.clim_slider_update)
+        self.layer.events.clim.connect(lambda e: self.clim_slider_update())
```

Comment:
Can you explain why this is needed?

Reply:
I dropped the 2nd `event` argument for `clim_slider_update` as it wasn't used, but the callback expects it. we use this pattern frequently in the code base

Reply:
Maybe add "event=None" to the signature instead?

Reply:
I prefer having the function call be simpler (i.e. no unnecessary arg). I think @kne42 has advocated for this in the past too. Thoughts @jni?

Reply:
@sofroniewn I prefer having code that can run without lambdas, *and* unified callback signatures, but I'll leave the discussion for later.

Reply:
ok that's fine, we'll standardize in a follow up PR